examples/kubernetes/README.md

```diff
@@ -25,7 +25,7 @@ Next to Prometheus, Alertmanager, node-exporter, prometheus-adapter and Grafana,
 git clone git@github.com:prometheus-operator/kube-prometheus.git
 cd kube-prometheus
 # Deploy the CRDs and the Prometheus Operator
-kubectl apply -f ./manifests/setup
+kubectl apply --server-side=true -f ./manifests/setup
```

Comment:
Why do we need to change this to server side apply?

Reply:
The CRD manifests are too large to be applied client-side. The same approach is used in the README to install the Pyrra CRDs and it's a common pattern to solve this problem.